laravel-app/app/Services/PostService.php

```diff
@@ -25,12 +23,8 @@ public function createPost(User $user, array $validatedData): UserPost
 
     /**
      * To update an existing post.
-     *
-     * @param UserPost $post
-     * @param array<string, mixed> $validatedData
-     * @return bool
      */
-    public function updatePost(UserPost $post, array $validatedData): bool
+    public function updatePost(UserPost $post, array $validatedData): bool // @phpstan-ignore-line
```

Comment:
why is there phpstan-ignore-line here??

Reply:
sir the one you commented is the outdated version of the PR, I already removed them from last 2 commits. Thank you